contents/docs/experiments/experiment-significance.mdx

```diff
@@ -16,10 +16,7 @@ Below are all the formulas and calculations we use to determine the significance
 
 In the field of experimentation, there are two primary statistical approaches: frequentist and Bayesian.
 
-We adopt the Bayesian methodology because it directly answers the question: "Is variant A better than variant B?" This approach minimizes judgment errors, which are more common with the frequentist method.
-
-> In a frequentist approach, you start with a null hypothesis, which typically represents the current state of things or no effect. For example, the null hypothesis might state that there is no difference between variant A and variant B. 
```

Comment:
Should we mention the Frequentist approach somewhere else if we're dropping it here?

Reply:
Hmm I'll add a link somewhere which compares the frequentist vs bayesian. I dont really want to explain in the docs, since its not super relevant

Reply:
> I dont really want to explain in the docs, since its not super relevant

Tell me more?

Reply:
Well, we're talking about what we mean when it comes to experiment signficance in this docs. The main bit is how we calculate it for our different trends/paths.

What frequentist is isn't relevant to this (highlighted by the fact that we put it in a callout :) ), so it's not really worth including.